sparse/usr/sbin/flash-partition

```diff
@@ -144,8 +144,17 @@ if on_device; then
         # If we are flashing boot partition and we are running multirom setup
         # which can be detected by having data_subdir= in th ecmdline skip boot
         # partition flashing as it is not needed.
-        if test $1 = boot && grep data_subdir= /proc/cmdline &> /dev/null; then
+        mrom_check=$(grep data_subdir= /proc/cmdline)
```

Comment:
originally was the mrom_path extracted before the if and used in -d. sledges objected as a sed being to slow...